modules/migration-mtc-release-notes-1-7.adoc

```diff
@@ -0,0 +1,27 @@
+
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/mtc-release-notes.adoc
+:_content-type: REFERENCE
+[id="migration-mtc-release-notes-1-7_{context}"]
+= {mtc-full} 1.7 release notes
+
+[id="new-features-and-enhancements-1-7_{context}"]
+== New features and enhancements
+
+This release has the following new features and enhancements:
+
+* The {mtc-full} ({mtc-short}) operator now depends upon the OpenShift API for Data Protection (OADP) operator. When you install the {mtc-short} operator, the Operator Lifecycle Manager (OLM) automatically installs the OADP operator in the same namespace.
+
+* You can migrate from a source cluster that is behind a firewall to a cloud-based destination cluster by establishing a network tunnel between the two clusters using the `crane tunnel-api` command.
+
+* Converting storage classes in the MTC web console: You can convert the storage class of a persistent volume (PV) by migrating it within the same cluster.
+
+[id="known-issues-1-7_{context}"]
+== Known issues
+
+This release has the following known issues:
+
+* On {product-title} 3.10, the `MigrationController` pod takes too long to restart. The Bugzilla report contains a workaround. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1986796[*BZ#1986796*])
+* `MigPlan` custom resource does not display a warning when an AWS gp2 PVC has no available space. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1963927[*BZ#1963927*])
+* Block storage for IBM Cloud must be in the same availability zone. See the link:https://cloud.ibm.com/docs/vpc?topic=vpc-block-storage-vpc-faq[IBM FAQ for block storage for virtual private cloud].
```

Comment:
Consider specifying what also needs to be in the same availability zone. I assume that it's the cluster, but I am not sure based on the current context.

Reply:
Will not fix. I hope this isn't high priority.